instrument/loq/sans.py

```diff
@@ -150,43 +130,39 @@ def changer_pos(self, pos):
     @staticmethod
     def set_aperture(size):
         if size == "":
+            print("Aperture unchanged")
             pass
-        elif size.upper() == "SMALL":
-            gen.cset(Aperture_2="SMALL")
-        elif size.upper() == "MEDIUM":
-            gen.cset(Aperture_2="MEDIUM")
-        elif size.upper() == "LARGE":
-            gen.cset(Aperture_2="LARGE")
+        elif size.upper() in ["SMALL", "MEDIUM", "LARGE"]:
+            gen.cset(Aperture_2=size.upper())
         else:
-            raise RuntimeError("Slit size {} is undefined".format(size))
+            raise RuntimeError(f"Slit size {size} is undefined")
 
     def _detector_is_on(self):
         """Is the detector currently on?"""
         return self.get_pv("MOXA12XX_02:CH0:AI:RBV") > 2
 
-    @staticmethod
-    def _detector_turn_on(delay=True):
+    def _detector_turn_on(self, delay=True):
         raise NotImplementedError("Detector toggling is not supported LOQ")
         # for x in range(8):
         #     self.send_pv("CAEN:hv0:4:{}:pwonoff".format(x), "On")
 
-    @staticmethod
-    def _detector_turn_off(delay=True):
+    def _detector_turn_off(self, delay=True):
         raise NotImplementedError("Detector toggling is not supported on LOQ")
         # for x in range(8):
         #     self.send_pv("CAEN:hv0:4:{}:pwonoff".format(x), "Off")
 
     def _configure_sans_custom(self):
+        # Set Aperture_2?
         gen.cset(Tx_Mon="OUT")
         gen.waitfor_move()
 
     def _configure_trans_custom(self):
-        gen.cset(Aperture_2="SMALL")
+        self.set_aperture("SMALL")
         gen.cset(Tx_Mon="IN")
         gen.waitfor_move()
 
     # pylint: disable=invalid-name
-    def J1(self, temperature_1, temperature_2):
+    def run_off_julabo_1(self, temperature_1, temperature_2):
```

Comment:
Were the instrument scientists okay with this name change?  The J1 and J2 names were chosen because it was what they had used before.  I'm happy with the new name if they're okay with it.

Reply:
To the best of my knowledge the SANS Group was not consulted about this change!